custom-connectors/Public360/Artifacts/apiProperties.json

```diff
@@ -0,0 +1,63 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "4457e12640df-8aea-454e-ab40-bfaa70ae",

```

Comment:
The client ID should be removed and replaced with a dummy value or `null` for the purposes of open sourcing